### ISSUE TITLE
follow-up

### ISSUE THREAD

**Issue by cafernandezlo** [MALL-Lab/drugEnrichment#1]
## Project description

In YACCS we run Drug Enrichment Analysis for 1 signature and only in CRC cell lines in only one drug repurposing dataset (PRISM).

In addition, YACCS is hard to interpret biologically.

So, can we extend this analysis simply runing the same approach but with all the GRNs (i.e. using CollecTRI)? In that case if a signature is correlated with a MoA, we can go deeper biologically speaking. In addition, we can see relationships in specific cancer types and also across cancer types. Is a signature general to any cancer?

Also, we should include more datasets: PRISM, (ask to Seoane and check YACCS github)

Other idea is instead to group drugs according MoA, we can clusterize them based on other things (Pau´s and Barabara´s work).

## Project Folder

Practicas: https://udcgal.sharepoint.com/:f:/s/MALL-lab/EiOxvXbMljhHmbfhn_oW7AsBfYoHNUcgv1pZsZCch2ZVuA?e=9zoVXu
TFM: https://udcgal.sharepoint.com/:f:/s/MALL-lab/EnAke0yXi41Og2xEBDg6QE4Bu3HGLcz_AYlKGGpCMOxKQA?e=4rdMGx

## Labnotebook
see [here](https://udcgal.sharepoint.com/:w:/r/sites/MALL-lab/Documentos%20compartidos/General/Trainees/TFM_MUBICS_2024_SamuelMartin/drugEnrichment_followup.docx?d=w84ba9a0420254d7b98aee013f175a366&csf=1&web=1&e=T6lqc2)

## Overleaf
https://www.overleaf.com/2717945114bcdnjwwqykmp#70b98a

## References

https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05343-8
https://www.nature.com/articles/nchembio.1986
https://www.nature.com/articles/s41698-024-00583-0
https://www.nature.com/articles/nbt.2877

**Comment by jlinaresb** [comment]
Lo mismo aquí @Samuel-Martin-Revuelta  confirma que todo está accesible en el one drive. Añade también el .zip del overleaf en la carpeta

**Comment by jlinaresb** [comment]
@cafernandezlo @DiegoFE94  cuidado porque os habeis cargado este issue del projecto donde teníamos el seguimiento

**Comment by jlinaresb** [comment]
@Samuel-Martin-Revuelta  @cafernandezlo 

I just moved this card from Trainees to Project. Let´s continue working on this. Basically the future development is:

* [x] Continue working in EACR history in case we have to present a poster
* [ ] Extend the analysis to run with all cell lines
* [ ] Extend the analysis in better clusterize drugs
* [ ] Try to merge forces with Seoane´s lab

**Comment by jlinaresb** [comment]
@Samuel-Martin-Revuelta do we have more results that we should discuss? how that is going?

**Comment by Samuel-Martin-Revuelta** [comment]
@jlinaresb Sorry, I've been working those days on Exprela's work, I'd like to keep working on this but maybe is better to wait one or two weeks just to get started with the other project. Also I think it would be a good idea arrange an express meeting since after trying the NMF with all cell lines I'm not sure how we should continue.

**Comment by jlinaresb** [comment]
Thanks for the update @Samuel-Martin-Revuelta !

Ok, sounds good. But maybe we can take advantage of the labmeetings to share the ideas with te people. What about to upload a card in the following weeks?

**Comment by jlinaresb** [comment]
I would say this project is done after EACR presentation. So, to minimize efforts I suggest merge this project with Sara´s project. So, shall we arrange a meeting during this month (or early next month) to organize it?

@cafernandezlo if you are interested as well, you are wellcome :)

**Comment by cafernandezlo** [comment]
Absolutely, and please remember ASEICA, we can focus on it. Shall we meet early?

Abstract submission deadline, 1 July 2025.

[Conference info](https://www.aseica.es/20th-aseica-international-congress)

**Comment by jlinaresb** [comment]
Yes, ASEICA is already in our minds. I think we should meet when @sgonzca has the results for the analysis she is doing. 

Then, I would like to brainstorm some ideas for next directions either between us or in a labmeeting.

**Comment by jlinaresb** [comment]
@cafernandezlo @sgonzca @Samuel-Martin-Revuelta 
Following this let´s have a meeting on June 20th to see Sara´s work status, decide ASEICA and define next steps. I´ll send you an invite
